### ISSUE TITLE
OOP

### ISSUE THREAD

**Issue by NoahSchiro** [Harrisonnad/PDF_Reader#1]
I often hear programmers talk about how people use object oriented programming in cases where it is totally unnecessary. Isn't this program simple enough that we could just format this as a function? Not super familiar with python, so I am not sure if there is some optimization to be had by making this structured around the Reader class

**Comment by Harrisonnad** [comment]
I agree with you whole heartedly but to run the program as a cli it needs
to be in an OOP format. Trust me I dont like over complicating anything as
much as the next guy. I believe in simplicity in design but unfortunately
as it it has to be this way.

**Comment by Harrisonnad** [comment]
When I expand the program to have a gui I think we can destructure it into
a simple function.
